### PR TITLE
GET /lecture/:id/questions/

### PR DESCRIPTION
**Pull request by mrkev**
*untested*

### CODE REVIEW

File: src/routers/GetQuestionsRouter.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  getPath(): string {
+    return '/lecture/:id/questions/';
+  }
+
+  async contentArray(req, pageInfo, error) {
+
+    const id = req.params.id
+    const quesitons = await QuestionsRepo
```
Comment (Jma353) [inline src/routers/GetQuestionsRouter.js]: `questions`

File: src/routers/GetQuestionsRouter.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// @flow
+import { Request } from 'express';
+import AppDevEdgeRouter from '../utils/AppDevEdgeRouter';
+import constants from '../utils/constants';
+import QuestionsRepo from '../repos/QuestionsRepo'
+
+import type { APIQuestion } from './APITypes'
```
Comment (Jma353) [inline src/routers/GetQuestionsRouter.js]: `type` imports should go above module imports.  This is a preference thing but I wanna try and standardize alphabetizing and ordered imports

File: src/routers/GetQuestionsRouter.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// @flow
+import { Request } from 'express';
```
Comment (Jma353) [inline src/routers/GetQuestionsRouter.js]: Alpha

File: src/routers/GetQuestionsRouter.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    return quesitons
+      .filter(Boolean)
+      .map(question => ({
+        node: {
+          id: question.id,
+          text: question.text,
+          type: question.type,
+          data: question.data,
+        },
+        cursor: question.createdAt.valueOf(),
```
Comment (Jma353) [inline src/routers/GetQuestionsRouter.js]: Cursor is a date?

Comment (mrkev) [reply src/routers/GetQuestionsRouter.js]: yes

**Pull request merged**